lib/synapse/service_watcher/zookeeper_aurora.rb

```diff
@@ -0,0 +1,76 @@
+# encoding: utf-8
+#
+# Variant of the Zookeeper service watcher that works with Apache Aurora
+# service announcements.
+#
+# Parameters:
+#   hosts: list of zookeeper hosts to query (List of Strings, required)
+#   path: "/path/to/serverset/in/zookeeper" (String, required)
+#   port_name: Named service endpoint (String, optional)
```

Comment:
Serversets allow you to register multiple ports for your service. For example, you can say that your service has an `http`, an `https`and a `admin` port. This option can be used to specify which port shall actually be used by synapse.